adventureGame/AdventureGame/src/Main.py

```diff
@@ -125,4 +130,72 @@ def constructDungeonRoom():
         # Print for debug.
         dungeonRoom1.printDungeonRoomInfo()
 
+def UserInput():
+    # Summary: get user input, and return all lower case
+    userinput = input("Please type a command here:: ")
+    return userinput.lower()
+
+def SplitInput(funcInput):
+    # Summary: split input string into seperate words, return array of words
+    split = funcInput.split() #returns an array
+    return split
+
+def CheckAction(funcInput):
+    # Summary: check that the command is valid
+    # This is done by splitting the command into two chunks, command word and target
+    # It will check command word first, then object dependant on the command
+    # ["walk", "hit", "inventory", "open"] -> command words that are accepted
+    # ["skeleton", "goblin", "kobold", "zombie"] -> objects that are accepted
+    acc_enemy = False #@ acc_enemy -> acceptable enemy - control variable for if the enemy is acceptable
+    accepted = False #@ accepted -> overall control variable for if the whole command is acceptable
+    enemyID = ""     #@ enemyID -> string variable to store enemy name
+    commandID = ""   #@ commandID -> string variable to store command name
+
+    try:
+        commandID = funcInput[0]
+    except:
+        print("You havent typed in a command!")
+    else:
+        # check that the command word (word 1) is in the list of commands
+        for i in range (0, len(acceptableAnswers)):
+            if (funcInput[0] == acceptableAnswers[i]):
+                accepted = True
+
+        # if the command is hit, check that the enemy exists
+        if (commandID == "hit"):
+            # check that there is an enemy to hit
+            try:
+                enemyID = funcInput[1]
+            except:
+                print("You need to say what you are hitting!")
+            else:
+                for i in range (0, len(dungeonEnemies)):
+                # word 2 is the enemy identifier
+                    if (enemyID == dungeonEnemies[i]):
+                        acc_enemy = True
+
+            # if the enemy does not exist, the command is invalid
+            if (acc_enemy == False):
+                accepted = False
+
+    return accepted
+
+def playerTurnCommand():
+    # Summary: Get user input, evaluate, loop if the input is not an acceptable command
+    # Do not allow the turn to run unless the command is correct
+    actionAccepted = False  #@ actionAccepted -> loop control variable
+
+    while (actionAccepted == False):
+        if inputTutorialFlag == False:
+            print("What do you wish to do? you can either: 'move', 'open' a chest, check 'inventory' or 'hit [enemy name]'")
+            #make instructions never print again.
+            inputTutorialFlag = true
```

Comment:
true statement needs to be capital letter to be syntactically correct. Please change to "True"

Reply:
good work